modules/targetAcquisition/targetAcquisition.py

```diff
@@ -43,10 +44,23 @@ def __init__ (self, deckLinkFrame=np.zeros((416, 416, 3))):
         deckLinkFrame : np.ndarray, optional
             Variable size array containing data about a video frame (as given by cv2.imread())
         """
-        self.boxes = [] # Contains BoundBox objects (see utils.py), each of which contains opposite corners of a rectangle by percentage of height and width of the image as (xmin, ymin) to (xmax, ymax)
+        self.boxes = []  # Contains BoundBox objects (see utils.py), each of which contains opposite corners of a rectangle by percentage of height and width of the image as (xmin, ymin) to (xmax, ymax)
         self.tentCoordinates = dict()
-        self.currentFrame = deckLinkFrame
-    
+        self.currentFrame = np.zeros(2)
```

Comment:
What's the purpose of initializing the frame to a size [2] array?

Reply:
Oh lmao this was for testing, it should be an empty array